src/mlpack/methods/linear_regression/linear_regression_main.cpp

```diff
@@ -140,10 +140,10 @@ int main(int argc, char* argv[])
     {
       // The initial predictors for y, Nx1.
       Timer::Start("load_responses");
-      responses = std::move(CLI::GetParam<vec>("training_responses"));
+      responses = CLI::GetParam<vec>("training_responses").t();
```

Comment:
We can change the definition of `training_responses` from `PARAM_COL_IN` to `PARAM_ROW_IN`, then the `.t()` isn't necessary.